tests/acceptance/test_project_settings_sampling.py

```diff
@@ -71,6 +72,7 @@ def wait_until_page_loaded(self):
         self.browser.get(self.path)
         self.browser.wait_until_not('[data-test-id="loading-indicator"]')
 
+    @pytest.mark.skip("We need to generate transactions from supported SDK")
```

Comment:
what happens if we don't have this skip?

Reply:
They fail because the "Next" button never gets enabled.
I created a follow-up ticket with more details here: https://getsentry.atlassian.net/browse/TET-351